### PR TITLE
Update custom_metrics.md

### PR DESCRIPTION
**Pull request by nicholas-devlin**
it's top 5000 now

image

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/metrics/custom_metrics.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If a metric is not submitted from one of the [more than {{< translate key="integ
 
 A custom metric is identified by **a unique combination of a metric's name and tag values (including the host tag)**. In general, any metric you send using [DogStatsD][3] or through a [custom Agent Check][4] is a custom metric.
 
-**Note**: Users with the Datadog Admin roles can see the monthly average number of custom metrics per hour and the top 500 custom metrics for their account in the [usage details page][5]. Learn more about [how custom metrics are counted][6].
+**Note**: Users with the Datadog Admin roles can see the monthly average number of custom metrics per hour and the top 5000 custom metrics for their account in the [usage details page][5]. Learn more about [how custom metrics are counted][6].
```
Comment (buraizu) [inline content/en/metrics/custom_metrics.md:31]:
```suggestion
**Note**: Users with the Datadog Admin role or `usage_read` permission can see the monthly average number of custom metrics per hour and the top 5000 custom metrics for their account in the [usage details page][5]. Learn more about [how custom metrics are counted][6].
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just made one minor copy suggestion and added detail about the specific permission, since permissions are offered at a more granular level now. I'm open to the idea of linking to the permissions page, but let me know what you think:
https://docs.datadoghq.com/account_management/rbac/permissions

File: content/en/metrics/custom_metrics.md (hunk 1 above)
Comment (draav) [reply content/en/metrics/custom_metrics.md:31]: cool looks good

**Comment by buraizu** [comment]
Verified in local build.

**Review by buraizu — APPROVED** [review]

**Pull request merged**